inc/admin.php

```diff
@@ -31,23 +31,10 @@
  * @return void
  */
 function setup() {
-	add_action( 'admin_enqueue_scripts', __NAMESPACE__ . '\enqueue_admin_assets' );
```

Comment:
So how are assets being enqueued now?

Reply:
`admin-compiled.css` file contained only few lines of CSS for notifications, I moved those styles to inline https://github.com/xwp/material-design-wp-theme/pull/165/files#diff-93d36a6d7cfdc8b27f5ec6e8d4959420e99295ddf46708ee6fd18489c9ccaf24R92